docs/build/tutorials/platform/create-a-local-test-network.md

```diff
@@ -151,3 +155,41 @@ That’s it! Your local version of Avalanche is up and running. It has the defau
 
 You can add more nodes to the network. Just remember to give unique values for `db-dir`, `http-port` , and `staking-port`.
 
+### Using the Avalanche Network Runner
+
+Install the [Avalanche Network Runner](../../tools/netrunner.md) by cloning the repo:
+
+```
+git clone https://github.com/ava-labs/avalanche-network-runner.git
+```
+
+As this is a `go` tool, we recommend installing according to the `$GOPATH` convention. The rest of this tutorial assumes this convention.
```

Comment:
reference to the GOPATH convertion? to find out how golang packages are usually installed into $GOPATH/src. There are some mkdir and chdir commands around the 'git clone' one.

Reply:
Personally I don't think that's required, but I'll let other people decide.

Reply:
For the clone option, you need to first create dir $GOPATH/src/github.com/ava-labs/ and then clone it in there. The doc may be more direct to follow if it tell that previous step.